docs/intro.rst

```diff
@@ -292,10 +292,35 @@ FromBytes
 Bool
 ....
 
+The checker test that a received value is a boolean type.
+
+    >>> t.Bool().check(True)
+    True
 
 StrBool
 .......
 
+If you need to check value that can be equivalent to a boolean type, you can use ``StrBool``.
+**Letter case doesn't matter.**
```

Comment:
I guess so it will be great to see when StdBool work. For that u can show all possible value for this checker.

```
('t', 'true', 'false', 'y', 'n', 'yes', 'no', 'on', 'off', '1', '0', 'none')
```